en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-guidelines.md

```diff
@@ -80,7 +80,25 @@ test your functional and non-functional requirements.
 
     WSO2 recommends that you perform the migration while the system is under minimum traffic. 
 
-13. When migrating a Kubernetes environment to a newer API Manager version, it is recommended to do the data migration in a separate VM, a local machine, or a single container. Once the data migration is complete, you can simply move the migrated databases into the containerized deployment in Kubernetes. 
+13. When attempting to migrate a distributed setup, you need to do the data migration for the Control Plane profile only. You can do the migration using the following command:
+
+    ```bash
+    sh api-manager.sh -Dprofile=control-plane -Dmigrate -DmigrateFromVersion=<product-version-number> -DmigratedVersion=4.1.0 -DrunPreMigration
+    ```
+
+14. Disable versioning in the registry configuration when migrating IS as a Key Manager from versions older than IS 5.9.0.
+
+    If there are frequently updating registry properties, having the versioning enabled for registry resources in the registry can lead to unnecessary growth in the registry related tables in the database. To avoid this, versioning has been disabled by default in API Manager 4.1.0.
+    
+    Therefore, if registry versioning was enabled in older versions of WSO2 API-M, it is **required** to run some scripts against **the database that is used by the registry**.
```

Comment:
no sense on some scripts. should be registry version disabling scripts. May be you can point the step 5 in a older documentation

Reply:
Fixed.